src/main/java/com/checkmarx/flow/CxFlowApplication.java

```diff
@@ -11,6 +12,7 @@
 @SpringBootApplication(scanBasePackages = { "com.checkmarx.sdk", "com.checkmarx.flow", "com.cx.restclient"})
 @EnableAspectJAutoProxy(proxyTargetClass = true)
 @EnableAsync
+@EnableCaching
```

Comment:
what is the purpose for this change?

Reply:
I used cache. I don't wanna to send many requests to gitlab. Check where I use anatation Cacheable

Reply:
I'm not sure what would be the impact of that. and I don't know how well it was tested
Can we remove this change from the PR?